Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicHandler.cpp

```diff
@@ -724,13 +729,13 @@ namespace AzToolsFramework
             }
 
             PrefabDom beforeState;
+            m_instanceDomGeneratorInterface->GenerateEntityDom(beforeState, *entity);
```

Comment:
until we have better error handling, it might at least be a good idea to return an AZ::Failure here if the function returns false.